packages/types/src/Metadata/v5/latest.substrate.v5.json

```diff
@@ -2,2575 +2,3503 @@
+              },
+              "fallback": "0x00",
+              "documentation": [
+                " Events deposited for the current block."
+              ]
+            },
+            {
+              "name": "EventCount",
+              "modifier": "Default",
+              "type": {
+                "PlainType": "EventIndex"
+              },
+              "fallback": "0x00000000",
+              "documentation": [
+                " The number of events in the `Events<T>` list."
+              ]
+            },
+            {
+              "name": "EventTopics",
+              "modifier": "Default",
+              "type": {
+                "DoubleMapType": {
+                  "hasher": "Blake2_256",
+                  "key1": "Null",
+                  "key2": "Hash",
+                  "value": "Vec<(BlockNumber,EventIndex)>",
+                  "key2Hasher": "Blake2_256"
+                }
+              },
+              "fallback": "0x00",
+              "documentation": [
+                " Mapping between a topic (represented by T::Hash) and a vector of indexes",
+                " of events in the `<Events<T>>` list.",
+                "",
+                " The first key serves no purpose. This field is declared as double_map just",
+                " for convenience of using `remove_prefix`.",
+                "",
+                " All topic vectors have deterministic storage locations depending on the topic. This",
+                " allows light-clients to leverage the changes trie storage tracking mechanism and",
+                " in case of changes fetch the list of events of interest.",
+                "",
+                " The value has the type `(T::BlockNumber, EventIndex)` because if we used only just",
+                " the `EventIndex` then in case if the topic has the same contents on the next block",
+                " no notification will be triggered thus the event might be lost."
+              ]
             }
-          },
-          "fallback": "0x0000000000000000",
-          "documentation": [" Extrinsics nonce for accounts."]
-        }, {
-          "name": "ExtrinsicCount",
-          "modifier": "Optional",
-          "type": {
-            "PlainType": "u32"
-          },
-          "fallback": "0x00",
-          "documentation": [" Total extrinsics count for the current block."]
-        }, {
-          "name": "AllExtrinsicsLen",
-          "modifier": "Optional",
-          "type": {
-            "PlainType": "u32"
-          },
-          "fallback": "0x00",
-          "documentation": [" Total length in bytes for all extrinsics put together, for the current block."]
-        }, {
-          "name": "BlockHash",
-          "modifier": "Default",
-          "type": {
-            "MapType": {
-              "hasher": "Blake2_256",
-              "key": "BlockNumber",
-              "value": "Hash",
-              "isLinked": false
+          ],
+          "calls": null,
```

Comment:
Something is SERIOUSLY wrong here - https://github.com/paritytech/substrate/blob/master/node/runtime/src/lib.rs#L250

Calls are exposed and do not appear here (e.g. new setCode)

Reply:
Logged a substrate issue for this - as soon as that is available, need to update 

(The bottom-line is that the changes in Method/extrinsics here would need to be re-adjusted when system does return calls, i.e. that will become the first index and everything else will move down, i.e. this would be bumped by 1 https://github.com/polkadot-js/api/pull/1016/files#diff-fdb8bb0e6f1aabc8679a048b348e91adR20)